eng/Versions.props

```diff
@@ -279,6 +279,7 @@
     <XunitAssertPackageVersion>$(XunitVersion)</XunitAssertPackageVersion>
     <XunitExtensibilityCorePackageVersion>$(XunitVersion)</XunitExtensibilityCorePackageVersion>
     <XunitExtensibilityExecutionPackageVersion>$(XunitVersion)</XunitExtensibilityExecutionPackageVersion>
+    <XUnitRunnerVisualStudioVersion>2.4.3</XUnitRunnerVisualStudioVersion>
```

Comment:
Teensy nit: Should we have a comment reminding us to remove this line once we pick up Arcade bits containing the dotnet/arcade#5908 fix❔

Reply:
Done: https://github.com/dotnet/aspnetcore/issues/24750